interface.go

```diff
@@ -253,6 +253,8 @@ type Pmax interface {
 	GetDirectorIDList(ctx context.Context, symID string) (*types.DirectorIDList, error)
 	// GetPortList returns a list of all the ports on a specified director/array.
 	GetPortList(ctx context.Context, symID string, directorID string, query string) (*types.PortList, error)
+	// GetPortListForSymmetrix returns a list of all the ports on a specified Symmetrix array.
+	GetPortListForSymmetrix(ctx context.Context, symID string, protocol string) (*types.PortList, error)
```

Comment:
Rename to `GetPortListByProtocol` so it's clear that this function is filtering portlist by protocol

Reply:
Thank you for the feedback. I have renamed the method to GetPortListByProtocol to better reflect its functionality of filtering the port list by protocol.